playbooks/common/openshift-checks/README.md

```diff
@@ -36,8 +28,14 @@ against your inventory file. Here is the step-by-step:
     $ cd openshift-ansible
     ```
 
-2. Run the playbook:
+2. Run the appropriate playbook:
+
+    ```console
+    $ ansible-playbook -i <inventory file> playbooks/common/openshift-checks/pre-install.yml
+    ```
+
+    or
 
     ```console
-    $ ansible-playbook -i <inventory file> playbooks/byo/openshift-preflight/check.yml
+    $ ansible-playbook -i <inventory file> playbooks/common/openshift-checks/health.yml
```

Comment:
I'm not against having playbooks under `common/` but I don't understand how our minds shifted away from `byo/`.